### PR TITLE
Open PR Bundle - iOS and Android to v5

### PR DESCRIPTION
**Pull request by mysport12**
- This PR seeks to combine a few of the open PRs into one that can be reviewed all at once to bring both iOS and Android up to v5 SDKs

- Master was forked and #108 was merged in
- Then Android was stepped to v4 via #130 
- iOS was brought to v5 via #138 
- Then Android was brought to v5 via #142 

Shout outs to the original PR contributors: @fabriziomoscon @Pagebakers @lonnylot and @aniravi24 - You all did the heavy lifting. Just trying to help with the merge effort

I avoided the formatting changes wherever possible and avoided the call -> activeCall variable name change in iOS

** Note haven't had a chance to test this out yet as I am just getting started on a project that will be using this. But figured it was worth the effort to pitch in and help if possible.

### CODE REVIEW

**Comment by Pagebakers** [comment]
Awesome! I will test this next week.

**Comment by fabriziomoscon** [comment]
Thank you @mysport12 I had started working on merging already with this PR: https://github.com/hoxfon/react-native-twilio-programmable-voice/pull/144
where I have listed a number of tests and steps that I must go through to complete the merge
I will look at the difference between my work and your 
a2e2c33

**Comment by mysport12** [comment]
Make sure you have my latest commit that includes a missed changed in the podspec.

File: android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java [hunk 1]
```diff
@@ -722,53 +759,46 @@ public void setSpeakerPhone(Boolean value) {
...
-         * required to be in this mode when playout and/or recording starts for
-         * best possible VoIP performance. Some devices have difficulties with speaker mode
-         * if this is not set.
-         */
-        audioManager.setMode(AudioManager.MODE_IN_COMMUNICATION);
-    }
+    private void setAudioFocus(boolean setFocus) {
+        if (audioManager != null) {
+            if (setFocus) {
+                savedAudioMode = audioManager.getMode();
```
Comment (imajindev) [inline android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java]: this savedAudioMode variable is not yet initialized

Comment (fabriziomoscon) [reply android/src/main/java/com/hoxfon/react/RNTwilioVoice/TwilioVoiceModule.java]: Thank you for testing, could you please review #144 ?
That is the PR which I finally intend to merge

**Pull request closed without merging**